lib/ProductOpener/DataQualityFood.pm

```diff
@@ -1180,6 +1206,7 @@ sub check_quality_food($) {
 	check_categories($product_ref);
 	compare_nutriscore_with_value_from_producer($product_ref);
 	check_ecoscore_data($product_ref);
+	check_food_groups($product_ref);
```

Comment:
Just a comment to say I'm very happy with this kind of programming: small function with a clear scope that we stack together, documented function and import with the name of imported functions (no need for ProductOpener libs, but for external libs, it helps a lot !)